packages/frontend-2/components/viewer/explorer/Filters.vue

```diff
@@ -2,19 +2,19 @@
   <ViewerLayoutPanel class="mt-2" hide-close>
     <template #title>Filtering</template>
     <template #actions>
-      <div class="flex justify-between items-center w-full">
-        <div class="-mt-1">
-          <FormButton
-            v-tippy="'Change Filter'"
-            text
-            size="xs"
-            :icon-right="showAllFilters ? ChevronUpIcon : ChevronDownIcon"
-            class="capitalize"
-            @click="showAllFilters = !showAllFilters"
-          >
+      <div class="flex justify-between items-center w-full gap-2">
+        <FormButton
+          v-tippy="'Change Filter'"
+          text
+          size="xs"
+          :icon-right="showAllFilters ? ChevronUpIcon : ChevronDownIcon"
+          class="capitalize"
```

Comment:
I think you also need a `truncate` class on the button. The reset + coloring buttons do a small jump to the right when the button text is too long. Gif:
![CleanShot 2024-05-14 at 13 25 45](https://github.com/specklesystems/speckle-server/assets/2694102/773e8c21-7f23-4d30-a1ba-e232d20a391e)